code/modules/mob/transform_procs.dm

```diff
@@ -82,14 +82,11 @@
 	O.notify_ai(1)
 
 	if(O.mind && O.mind.assigned_role == "Cyborg")
-		if(O.mind.role_alt_title == "Android")
+		if(O.mind.role_alt_title == "Android" || O.mind.role_alt_title == "Robot")
```

Comment:
I'd prefer android title just be axed, entirely.

Reply:
Why? It seems like a redundant title to me, so I don't object to axing it, but when making a request it is a good idea to explain your reason.